code/game/machinery/vending.dm

```diff
@@ -1572,6 +1572,7 @@ var/global/num_vending_terminals = 1
 		/obj/item/weapon/reagent_containers/syringe/antiviral = 4,
 		/obj/item/weapon/reagent_containers/syringe = 12,
 		/obj/item/device/healthanalyzer = 5,
+		/obj/item/device/antibody_scanner = 5,
```

Comment:
do we WANT antibody scanners to become super common? including the others this means on box there will be 18 roundstart antibody scanners (not counting the ones being built into the bodyscanner as a result of your changes), quite tedious to get rid of them all if antag viro

Reply:
To answer the question yes I want it to become a common tool to use in medbay.

These let you know the strength of unidentified pathogen, and their antigen if you scan repeatedly after injecting spaceacillin, but even knowing that is useless if you don't have access to vaccine. If the traitor virologist destroyed Viro and never distributed any cure, it'll be R&D's or an ERT's job to help create vaccines.

I find it funny how some people come tell me that new viro makes it too easy for antags to viruloose, while others tell me that it makes it too hard for them. I guess we'll see ultimately.

Having the tools is only half the work, but I think at least this tool should be readily available.